src/config/models.rs

```diff
@@ -61,6 +61,10 @@ pub struct ConfigOptsServe {
     #[structopt(long = "proxy-ws")]
     #[serde(default)]
     pub proxy_ws: bool,
+    /// Whether to auto-reload the web page when a build completes.
+    #[structopt(long)]
+    #[serde(default)]
+    pub autoreload: bool,
```

Comment:
Let's definitely make this default to `true`. Folks are going to want it enabled far more often than not.